scripts/build

```diff
@@ -6,7 +6,7 @@ source $(dirname $0)/version
 LINKFLAGS="-X main.Version=$VERSION
            -X main.GitCommit=$GITCOMMIT
            -X main.BuildDate=$BUILDDATE
-           -linkmode external -extldflags -static"
+            -extldflags -static -s"
```

Comment:
The indent looks off.

Reply:
What is the `-s` for?

Reply:
`Remove all symbol table and relocation information from the executable.` 
Same as https://github.com/longhorn/longhorn-manager/blob/master/scripts/build#L9

Ref: https://gcc.gnu.org/onlinedocs/gcc/Link-Options.html

Reply:
I am curious how with or without the `-s` would impact the share-manager. Or are we adding this for consistency?

Reply:
Just for consistency.
It is like `strip` that removes the symbols which are usually for debugging tools such as gdb.